backend/pkg/controllers/main.controller.go

```diff
@@ -681,6 +680,69 @@ func (c *MainController) RefreshTokenHandler(ctx *gin.Context) {
 	})
 }
 
+func (c *MainController) AuthenticateAgent(ctx *gin.Context) {
+	var user models.User
+	userId, err := getUserIdFromToken(ctx)
```

Comment:
please add an empty line to separate the variables declaration from the function body